src/main/java/fsca.java

```diff
@@ -0,0 +1,3 @@
+public class fsca {
+
+}
```

Comment:
you have used class with lowercase please change it to Uppercase

Reply:
ok